imports/pages/give/review/index.js

```diff
@@ -104,15 +84,17 @@ export class Template extends Component {
           additionalMessage={
             "Please click 'Done' in the top left of your screen to get back to the app"
           }
+          goToStepOne={() => {}}
```

Comment:
@johnthepink @jbaxleyiii these props are required by flow and making them optional causes a series of cascading flow failures. Is there a better way to satisfy this prop?